iridium-core-server/src/test/java/software/iridium/api/TokenAuthenticationFilterTest.java

```diff
@@ -71,6 +70,23 @@ public void getPreAuthenticatedPrincipal_AllGood_BehavesAsExpected() {
     assertThat(principalUser.getName(), is(equalTo(username)));
   }
 
+  @Test
+  public void getPreAuthenticatedPrincipal_BasicAuth_BehavesAsExpected() {
+    final var accessTokenValue = "Basic theEncodedTokenValue";
+    final var exchangePath = "/oauth/token";
+
+    when(mockServletRequest.getHeader(eq(HttpHeaders.AUTHORIZATION))).thenReturn(accessTokenValue);
+
+    when(mockServletRequest.getServletPath()).thenReturn(exchangePath);
+
+    assertThat(subject.getPreAuthenticatedPrincipal(mockServletRequest), nullValue());
```

Comment:
Even the unit test is verifying the null value. . . . If this isn't the path I would expect an error message.